src/skin/firstRun.html

```diff
@@ -28,6 +28,17 @@ <h1 class="i18n_name"></h1>
       </div>
     </header>
 
+    <!-- arrow showing users where to pin their extension, only shown on chromium -->
+    <div id="pin-nudge-arrow">
+      <div>
+        <a href="" id="dismiss-nudge" role="button" aria-label="i18n_report_close">
+          <img src="../icons/close.svg" alt="">
```

Comment:
Why do we need a new close.svg when we already have a similar "X" to close elsewhere?

Reply:
close.svg is basically the same as the inline svg! I meant to replace the inline svg with the re-usable icon, since it'll reduce lines of code and increase readability. is that okay with you?